terraform/ecs_ec2/daemon/main.tf

```diff
@@ -37,7 +37,7 @@ resource "aws_launch_configuration" "cluster" {
 
   user_data = "#!/bin/bash\necho ECS_CLUSTER=${aws_ecs_cluster.cluster.name} >> /etc/ecs/ecs.config"
   metadata_options {
-    http_endpoint               = "enabled"
+    http_endpoint               =  var.metadataEnabled
```

Comment:
How do we tell ecs metadata is up? Have we tested with each version of ecs metadata?